queue_job/wizards/queue_terminate_job_views.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="queue_terminate_job_view_form" model="ir.ui.view">
+        <field name="name">Terminate Jobs</field>
+        <field name="model">queue.terminate.job</field>
+        <field name="arch" type="xml">
+            <form string="Terminate Jobs">
+                <group string="The selected jobs will be terminated.">
```

Comment:
Maybe add a label explaining that those jobs will end up in status `failed` with an error message " cursor already closed".